### PR TITLE
Feature/vpc cni service account

### PR DESCRIPTION
**Pull request by shapirov103**
*Issue #, if available:*
Addresses #582 

*Description of changes:*
Adds dedicated IRSA account for VPC CNI and provides documentation on how to remove VPC CNI policy from the nodes. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by shapirov103** [comment]
@elamaran11 please review when you get a chance.

**Review by elamaran11 — DISMISSED** [review]
@shapirov103 LGTM. Great work in having IRSA for VPC CNI. Only minor feedback is verbiage.

File: docs/addons/vpc-cni.md [hunk 1]
```diff
@@ -97,6 +97,50 @@ const blueprint = blueprints.EksBlueprint.builder()
   .build(app, 'my-stack-name');
 ``` 
 
+## VPC-CNI Service Account and IRSA config
+
+VPC CNI add-on supports creation of an IRSA role for the add-on if the add-on configuration provides managed policies for the add-on.
```
[the remark this answers is not in the document]
Comment (shapirov103) [reply docs/addons/vpc-cni.md]: That meant to say if user passes managed policies then we create the IRSA. Otherwise we don't. 

ADDRESSED

**Review by elamaran11 — APPROVED** [review]
LGTM.

**Pull request merged**